src/blk/pmem/PMEMDevice.cc

```diff
@@ -35,7 +40,7 @@
 
 PMEMDevice::PMEMDevice(CephContext *cct, aio_callback_t cb, void *cbpriv)
   : BlockDevice(cct, cb, cbpriv),
-    fd(-1), addr(0),
+    fd(-1), addr(0), devdax_device(false),
```

Comment:
now that this variable is initialized in-class, could just drop this line.

Reply:
ack